### PR TITLE
Replaced extension(s) with a plural check

### PR DESCRIPTION
**Pull request by boynedmaster**
[UNTESTED, WOULD BE GREATLY APPRECIATED IF SOMEONE CHECKED IF THIS WORKED. IT SHOULD, SINCE I DIDN'T CHANGE TOO MUCH]
Replaced extension(s) with extension if there's only 1, and extensions if there's more than 1

### CODE REVIEW

**Comment by Wolvan** [comment]
This looks good to me but I have to discuss inline conditionals with the team before merge, some people don't really like it iirc

**Comment by ThePsionic** [comment]
Also make sure to bump the version.

**Comment by Wolvan** [comment]
@boynedmaster After some discussing we decided that an inline ternary operator is not allowed. Please create a variable and set it with an if/else statement before this code block

**Comment by boynedmaster** [comment]
Changed the ternary. Will up the version.

File: Extensions/xkit_updates.js [hunk 1]
```diff
@@ -153,13 +153,18 @@ XKit.extensions.xkit_updates = new Object({
 				var to_show = XKit.tools.get_setting("xkit_show_update_notifications","true");
 
 				if (to_show === "true") {
-
-					XKit.notifications.add("XKit updated " + XKit.extensions.xkit_updates.updated_list.length + " extension(s). Click here to view them.", "ok", true, function() {
+					var suffix = "";
+					
+					if(XKit.extensions.xkit_updates.updated_list.length != 1){
```
Comment (Wolvan) [inline Extensions/xkit_updates.js]: Use ``!==`` here, please

**Comment by nightpool** [comment]
@Wolvan was there a reason for that change? it wasn't something gulp was unhappy about, right?

**Comment by nightpool** [comment]
LGTM @homu r+

**Comment by Wolvan** [comment]
@nightpool Now, it wasn't gulp being unhappy, but we decided on having ``!=`` only if absolutely necessary

**Comment by nightpool** [comment]
cool! new homu features 💯

**Comment by nightpool** [comment]
@wolvan I hadn't heard about that—as far as I know === and !== are only
necessary in places where coercion can happen. (i.e. when comparing against
true or false, or the empty string, or undefined)

**Pull request merged**